tests/monotouch-test/ObjCRuntime/RegistrarTest.cs

```diff
@@ -75,6 +84,7 @@ public class RegistrarTest {
 			}
 		}
 
+#if !MONOMAC
```

Comment:
I don't remember now why I #ifed these out (which is why most places I commented out tests completely I tried to comment why I did. Apparently didn't here). Looking into it now.